docs/ui/auth/fragments/vue/auth-state-management.md

```diff
@@ -0,0 +1,33 @@
+```html
+<template>
+  <div>
+    <amplify-authenticator v-if="authState !== 'signedin'"></amplify-authenticator>
+    <div v-if="authState === 'signedin' && user">
+      <amplify-sign-out></amplify-sign-out>
+      <div>Hello, {{user.username}}</div>
+    </div>
+  </div>
+</template>
+```
+```js
```

Comment:
Could we say where this content needs to be put similar to how you have specified in angular. It will be easy for first timer framework users.

Reply:
Added a note to the top :D